.eslintrc

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "smashing-boxes/react"
+  "extends": "smashing-boxes/react",
+  "plugins": [
```

Comment:
This stuff should probably go in the eslint config: https://github.com/smashingboxes/eslint-config-smashing-boxes/blob/master/defaults.js